backup_and_restore/index.adoc

```diff
@@ -27,3 +26,39 @@ You might run into several situations where {product-title}  does not work as ex
 * You have lost the majority of your control plane hosts, leading to etcd quorum loss.
 
 You can always recover from a disaster situation by xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restoring your cluster to its previous state] using the saved etcd snapshots.
+
+[id="application-backup-restore-operations-overview"]
+== Application backup and restore operations
+
+As a cluster administrator, you can back up and restore applications running on {product-title} by using the OpenShift API for Data Protection (OADP).
+
+OADP backs up and restores Kubernetes resources and internal images, at the granularity of a namespace, by using a supported version, 1.7, of link:https://velero.io/[Velero]. See xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features].
+
+OADP backs up and restores PVs by using the cloud provider snapshot API, Container Snapshot Interface (CSI) snapshots, or link:https://restic.net/[Restic].
+
+You can install OADP with xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[default plugins].
+
+.OADP requirements
+
+OADP has the following requirements:
+
+* You must be logged in as a user with a `cluster-admin` role.
+* You must have object storage for storing backups:
+
+** Amazon Web Services
+** Microsoft Azure
+** Google Cloud Platform
+** Multicloud Object Gateway, a component of OpenShift Container Storage
+** S3-compatible object storage, for example, Noobaa or Minio
+
+:FeatureName: The `CloudStorage` API for S3 storage
+include::modules/technology-preview.adoc[]
+
+* You must have a cloud provider that has a native snapshot API or supports CSI snapshots if you want to back up PVs with snapshots:
+
+** Amazon Web Services
+** Microsoft Azure
+** Google Cloud Platform
+** CSI-enabled storage, for example, Ceph RBD or Ceph FS
```

Comment:
```suggestion
** CSI-enabled storage, such as Ceph RBD or Ceph FS
```
On first read, I saw " CSI-enabled storage, for example."

Reply:
To fit in the list, perhaps this should be reworded to something like this:

* any provider that uses CSI-enabled storage, such as Ceph RBD or Ceph FS

Reply:
OK. I can do that. I also saw that I forgot to mention that it's not just CSI but CSI snapshots, so I will fix that.